blueprints/addon/index.js

```diff
@@ -161,7 +161,23 @@ function alphabetizeDependencies(contents) {
 }
 
 function writeContentsToFile(contents, fileName) {
-  var packagePath = path.join(this.path, 'files', fileName);
+  var dirPath = path.join(this.path, 'files');
+  var packagePath = path.join(dirPath, fileName);
 
-  fs.writeFileSync(packagePath, stringifyAndNormalize(contents));
+  try {
+    fs.writeFileSync(packagePath, stringifyAndNormalize(contents));
```

Comment:
@kamilogorek although you PR improves the error message, I suspect we can get rid of the problem in most cases if we don't incorrectly write to this file in some scenarios.